renderer/views/app.js

```diff
@@ -23,7 +23,8 @@ function App (state, dispatch) {
     state.playing.mouseStationarySince !== 0 &&
     new Date().getTime() - state.playing.mouseStationarySince > 2000 &&
     !state.playing.isPaused &&
-    state.playing.location === 'local'
+    state.playing.location === 'local' &&
+      state.playing.playbackRate === 1
```

Comment:
Indentation.

This line makes it so that the player controls *always* show when the playback rate is !== 1

In addition, I think we should hide the playback rate display when the playback rate is === 1